tests/ui/double_parens.rs

```diff
@@ -23,7 +19,7 @@ impl DummyStruct {
 }
 
 fn simple_double_parens() -> i32 {
-    ((0))
```

Comment:
missing rustfmt_skip... or we just remove the lint entirely as it's covered by rustfmt?